Neos.Neos/Documentation/References/PropertyEditorReference.rst

```diff
@@ -92,12 +92,18 @@ In case the text input should span multiple lines, a ``TextAreaEditor`` should b
             group: 'document'
             editor: 'Neos.Neos/Inspector/Editors/TextAreaEditor'
             editorOptions:
-              rows: 7
+              minRows: 5
+              expandedRows: 10
+              maxRows: 20
 
 Options Reference:
 
-``rows`` (integer)
-	Number of lines this textarea should have; Default ``5``.
+``minRows`` (integer)
```

Comment:
In the other Options References, there is always a space between the options as far as I can see. Could you perhaps add the space for consistency?

Reply:
Thank you. Done.